demo/vue/app/frontend/src/components/base/BaseAuthButton.vue

```diff
@@ -30,7 +30,10 @@ export default {
     login() {
       if (this.keycloakReady) {
         window.location.replace(
-          this.createLoginUrl() + '&pres_req_conf_id=' + this.presReqConfId
+          this.createLoginUrl() +
+            '&pres_req_conf_id=' +
+            this.presReqConfId +
+            '&back_to_url=http://localhost:8080/',
```

Comment:
Hardcoded in simple demo app (like where we set the port). This is an example of where a calling app would provide environment-configured URL calling to VCAuth